org-code-javabuilder/lib/src/main/java/org/code/javabuilder/FatalErrorKey.java

```diff
@@ -0,0 +1,19 @@
+package org.code.javabuilder;
+
+import static org.code.javabuilder.LambdaErrorCodes.LOW_DISK_SPACE_ERROR_CODE;
+import static org.code.javabuilder.LambdaErrorCodes.TEMP_DIRECTORY_CLEANUP_ERROR_CODE;
+
+public enum FatalErrorKey {
+  LOW_DISK_SPACE(LOW_DISK_SPACE_ERROR_CODE),
```

Comment:
For my understanding, what is going on syntactically here?

Reply:
So because this enum has a constructor (below on line 12), each enum constant needs to be constructed with an int parameter. In this case, I'm just passing in integer constants from LambdaErrorCodes. This will be useful for when we need to call `System.exit(int code)` when we encounter a FatalError because we can just use the error code associated with the enum constant.